packages/nitro-protocol/gas-benchmarks/gas.ts

```diff
@@ -0,0 +1,53 @@
+type GasRequiredTo = Record<
+  Path,
+  {
+    vanillaNitro: number | any;
```

Comment:
```suggestion
    vanillaNitro: number | Record<string, number>;
```
(nit)

Reply:
Yeah I tried this before but it didn't seem to work! Will have another go

Reply:
977ef96